internal/tengo/outmessage.tengo

```diff
@@ -12,7 +12,7 @@ text := import("text")
 
 // start - strip irc colors 
 // if we're not sending to an irc bridge we strip the IRC colors
-if inProtocol == "irc" {
+if (inProtocol == "irc") && (outProtocol != "irc") {
```

Comment:
Not going to add a breaking change at this moment, you can make a PR for this which will be added in for v1.15.0
First going to make sure everything works before making more changes :)